src/app/examples/stock-market/stock-market.reducer.spec.ts

```diff
@@ -0,0 +1,93 @@
+import { Stock, StockMarketState } from './stock-market.model';
+import { HttpErrorResponse } from '@angular/common/http';
+import { stockMarketReducer, initialState } from './stock-market.reducer';
+import {
+  StockMarketActions,
+  ActionStockMarketRetrieve,
+  ActionStockMarketRetrieveError,
+  ActionStockMarketRetrieveSuccess
+} from './stock-market.actions';
+
+const originalState = {
+  symbol: 'LCLT',
+  loading: false,
+  stock: {
+    symbol: 'LCLT',
+    exchange: 'TestExchange',
+    last: '1.00',
+    ccy: 'USD',
+    change: '+0.05',
+    changePositive: true,
+    changeNegative: false,
+    changePercent: '+5%'
+  }
+} as StockMarketState;
```

Comment:
Could this be also `const originalState: StockMarketState = {}` because it has all the needed properties?

Reply:
I think they are essentially the same, but I agree that your suggestion reads better/the expected way.